services/api_gateway/pom.xml

```diff
@@ -87,6 +87,17 @@
             <version>0.1.0-SNAPSHOT</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
```

Comment:
We just merged new UTs to VPC and Route Controllers. Could you please add the docs accordingly. Thank you.

Reply:
Ok let me add it

Now there is a problem, most microservices mvn test need to connect to the database, I currently deploy the database can be executed, but it is best to be able to leave the database

Reply:
Is it possible to have a flag which could switch off DB access when generating the docs?

Reply:
The document is generated after the unit test is completed. As long as the unit test does not depend on the database during execution, the document can be generated.

Reply:
I see. Please work with @chenpiaoping to see if we can support DB-related UTs by Ignite mock.